golang/x/relay/client/cli/query.go

```diff
@@ -90,3 +91,58 @@ func GetCmdIsAncestor(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+func GetCmdFindAncestor(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		// what are the arguments. <> for required, [] for optional
+		Use:     "findancestor <digest> <offset>",
+		Example: "findancestor 12..ab 2", // how do you use it?
+		// a help message. shows on `help isancestor`
+		Long: "Finds the ancestor of a given digest",
```

Comment:
maybe `"Finds  the digest <offset> blocks before <digest>"`

also, should probably make a note on what it does if `digest` is unknown, or if the ancestor is unknown :thinking: